### PR TITLE
Improved German translation

### PR DESCRIPTION
**Pull request by DecodazTV**
just some locales update

### CODE REVIEW

File: locales/de.lua [hunk 1]
```diff
@@ -1,192 +1,192 @@
...
+  ['amount_invalid'] = 'ungültiger Betrag',
+  ['players_nearby'] = 'keine Spieler in der Nähe',
+  ['ex_inv_lim'] = 'aktion nicht möglich, Inventarlimit überschritten für ~y~%s~s~',
+  ['imp_invalid_quantity'] = 'aktion nicht möglich, ungültige Anzahl',
+  ['imp_invalid_amount'] = 'aktion nicht möglich, ungültiger Betrag',
+  ['threw_standard'] = 'you threw ~y~%sx~s~ ~b~%s~s~',
+  ['threw_money'] = 'du wirfst ~g~$%s~s~ ~b~Dollar~s~ weg',
+  ['threw_account'] = 'du wirfst ~g~$%s~s~ ~b~%s~s~ weg',
+  ['threw_weapon'] = 'du wirfst ~b~%s~s~ weg',
+  ['threw_weapon_ammo'] = 'du wirfst ~b~%s~s~ mit ~o~%sx %s~s~ Kugeln weg',
```
Comment (ElPumpo) [inline locales/de.lua]: You realize that the last `%s` is the proper ammo label?

Comment (DecodazTV) [reply locales/de.lua]: yeah i got it now fixed. idk what i though as i changed it.

File: locales/de.lua [hunk 2]
```diff
@@ -36,7 +36,7 @@ Locales['de'] = {
   ['threw_money'] = 'du wirfst ~g~$%s~s~ ~b~Dollar~s~ weg',
   ['threw_account'] = 'du wirfst ~g~$%s~s~ ~b~%s~s~ weg',
   ['threw_weapon'] = 'du wirfst ~b~%s~s~ weg',
-  ['threw_weapon_ammo'] = 'du wirfst ~b~%s~s~ mit ~o~%sx %s~s~ Kugeln weg',
+  ['threw_weapon_ammo'] = 'du wirfst ~b~%s~s~ mit ~o~%sx %s~s~ weg',
```
Comment (ElPumpo) [inline locales/de.lua]: Again I dont really know what the word `weg` is but have you tried dropping an weapon with ammo? That last `%s` in `threw_weapon_ammo` is the ammo label

File: locales/de.lua [hunk 3]
```diff
@@ -145,49 +145,49 @@ Locales['de'] = {
...
-  ['ammo_flaregun'] = 'flare(s)',
+  ['ammo_proxmine'] = 'mine(n)',
+  ['ammo_bzgas'] = 'can(n)',
+  ['ammo_ball'] = 'ball',
+  ['ammo_snowball'] = 'schneebälle',
+  ['ammo_flare'] = 'signalfackel(n)',
+  ['ammo_flaregun'] = 'signalfackeln(munition)',
 
   -- Weapon Tints
   ['tint_default'] = 'default skin',
```
Comment (ElPumpo) [inline locales/de.lua]: also translate tints

File: locales/de.lua (hunk 2 above)
Comment (DecodazTV) [reply locales/de.lua]: weg means something like "away" 
in this case you "you throw Carabine with 50 Bullets away"

File: locales/de.lua (hunk 3 above)
Comment (DecodazTV) [reply locales/de.lua]: ill do them now

**Comment by ElPumpo** [comment]
Thanks my friends for helping out!

**Pull request merged**